test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests.cs

```diff
@@ -1781,14 +1781,14 @@ public async Task RestoreCommand_CentralVersion_ErrorWhenNotAllPRItemsHaveCoresp
                 // Package Bar does not have a corresponding PackageVersion 
                 var packageRefDependecyBar = new LibraryDependency()
```

Comment:
typo - dependency

Reply:
This file only changed because of `dotnet restore` command.
Since it's not connected to my code logic better not to touch for now.